.github/workflows/test_frameworks.yml

```diff
@@ -26,11 +26,28 @@ jobs:
         continue-on-error: true
 
   bottle-testsuite-0_12_19:
+    strategy:
+      matrix:
+        include:
+          - suffix: Profiling
+            profiling: 1
+            iast: 0
+            appsec: 0
+          - suffix: IAST
+            profiling: 0
+            iast: 1
+            appsec: 0
+          - suffix: APPSEC
+            profiling: 0
+            iast: 0
+            appsec: 1
```

Comment:
Could we factor out each of these components like this?
```
profiles:
  profiling: &profiling
    suffix: Profiling
    profiling: 1
    iast: 0
    appsec: 0

  iast: &iast
    suffix: IAST
    profiling: 0
    iast: 1
    appsec: 0

  appsec: &appsec
    suffix: APPSEC
    profiling: 0
    iast: 0
    appsec: 1
```
And then use them like this?
```suggestion
# Reference the reusable items using aliases
strategy:
  matrix:
    include:
          - <<: *profiling
          - <<: *iast
          - <<: *appsec
```

Reply:
We could, if only github actions would support yaml anchors: https://github.com/actions/runner/issues/1182

Reply:
Ugh that's so tragic... 😞 It's been open for several _years_